src/styles/card.scss

```diff
@@ -0,0 +1,106 @@
+.card {
+  padding: 32px 0 16px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid $border-color;
+
+  &__img {
+    padding: 0 19px;
+    margin-bottom: 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__description {
+    padding: 0 16px;
+  }
+  &__title {
+    margin: 0;
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 18px; /* 150% */
+    color: $main-color;
+  }
+
+  &__code {
+    margin: 0;
+    margin-bottom: 16px;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px; /* 140% */
+    color: $secondary-color;
+  }
+
+  &__reviews {
+    padding: 0 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__price {
+    padding: 0 16px;
+    margin: 0;
+    margin-bottom: 16px;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__price-text {
+    margin: 0;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 18px; /* 150% */
+    color: $secondary-color;
+  }
+
+  &__price-number {
+    margin: 0;
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 18px; /* 112.5% */
+    color: $main-color;
+  }
+
+  &__buy-link {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-transform: uppercase;
+    text-decoration: none;
+    text-align: center;
+    color: $white-color;
+    margin: 0 16px;
+
+    width: 166px;
+    height: 40px;
```

Comment:
there is no need to use a fixed height for a button
because the height of the button is formed by its internal content - this is **the text** and **internal padding**